### ISSUE TITLE
Frozen in MacOS 12.2.1 (M1 Pro)

### ISSUE THREAD

**Issue by breezewish** [spencerwooo/PaimonMenuBar#2]
CPU keeps high and nothing shown in the menu bar:

<img width="504" alt="image" src="https://user-images.githubusercontent.com/1916485/165875365-418136fd-3ee3-4b20-ad1c-38d900e1b222.png">

<img width="2672" alt="image" src="https://user-images.githubusercontent.com/1916485/165875253-00eab786-726b-44ea-94c0-b119e5bdb241.png">

Chip: Apple M1 Pro

**Comment by breeswish** [comment]
> Unfortunately I have no idea what could be the problem... 😢
> 
> Do you have a Xcode environment to build and test the app locally?

I try to run it locally in Xcode and the frozen can be reproduced. I noticed that if the following are commented out then high CPU issue is relaxed (but no menu item will be displayed obviously):

```swift
        setupStatusItem()
        setupMenus()
```

Unfortunately I'm not familiar with Swift and that's what I can go so far. I will try to dig deeper when I'm free.

**Comment by breeswish** [comment]
So here is what I got so far. Actually this is combine with two issues:

1. CPU is high
2. Icon is not shown

I resolved the problem 2 by changing my menu bar cleaning app. Now I can see the menu bar item.

For the CPU high issue, I see some one reported similar problems in MacOS: https://www.youtube.com/watch?v=VufEEmuiAy8

I can reproduce the high CPU (and resolve the high CPU) by switching different wallpapers :/ Maybe we met similar issues here.
